### ISSUE TITLE
Replace jsonschema with avj

### ISSUE THREAD

**Issue by gajus** [trainiac/express-jsonschema#22]
jsonschema is incredibly slow compared to ajv (https://github.com/epoberezkin/ajv).

**Comment by vacekj** [comment]
Supporting this issue. Definitely can see some performance improvements by using ajv.

**Comment by gajus** [comment]
Yup, I've dropped `express-jsonschema` from production because of this issue.

**Comment by vacekj** [comment]
Does anyone have experience with [express-json-validator](https://www.npmjs.com/package/express-json-validator)? It uses ajv, but isn't that popular nor documented.

**Comment by vacekj** [comment]
Okay, I'm going to attempt to make a module similar to this that uses ajv. [Repo](https://github.com/JouzaLoL/express-json-validator-middleware)

**Comment by vacekj** [comment]
Module is up and running for those who want a faster alternative. Functionality is bare-bones at the moment, progress is being made as we speak!
[L I N K](https://github.com/JouzaLoL/express-json-validator-middleware)
Starring the repo is super welcome
